scripts/generate_psa_key_attributes.py

```diff
@@ -194,7 +275,7 @@ def main() -> None:
     parser.add_argument(
         "--key-from-file",
         help="Read key from PEM file",
-        type=argparse.FileType(mode="rb"),
+        type=str,
```

Comment:
Why did we change this to `str`?

Reply:
I was Lazy, I do not remember exactly what was the problem, but when I moved the generation code into a function so I could call it from the ncs_provision.py script I had some problem creating the argparse.FileType type when calling the generate_attr_file function.

Reply:
I changed it back to `argparse.FileType(mode="rb")`. And changed how it is called from `ncs_provision.py`